nexus_system_orchestrator/src/dispatch_transporter.hpp

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Double check if we need this.

Reply:
Actually yes! This is to execute this behavior tree node https://github.com/osrf/nexus/blob/c96fdbfb895c713e2a4282e81720613531f4d119/nexus_integration_tests/config/system_bts/main.xml#L8

What this system orchestrator node does is:

* Iterate over tasks in the work order
* Generate a transportation task for the current work order that visits all the workcells that have been assigned to each task.
* Bid this new transportation request to all the registered workcells. Fail if no workcell can do this transportation request.
* Output the transportation task as well as workcell that was assigned to it.

There is admittedly a bit of duplication in the bidding here and the bidding at the system orchestrator level. The problem that needs to be solved is that this specific task needs to have full knowledge of the result of the workcell task assignment to know where to send the AMRs, so it needs to run in a following step